src/state/reducers/childReducer.js

```diff
@@ -9,6 +9,7 @@ const initialState = {
   parentId: null,
   cohortId: null,
   memberId: null,
+  achievements: {},
```

Comment:
This might be better as an array of objects: [{},{}]
So that each achievement can be it's own object.